instrumentation/spring/spring-webflux-5.0/library/src/main/java/io/opentelemetry/instrumentation/spring/webflux/client/SpringWebfluxHttpClientTracer.java

```diff
@@ -97,4 +100,12 @@ private static MethodHandle findRawStatusCode() {
       return null;
     }
   }
+
+  // TODO cache this after
+  //  https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/1643
+  private static boolean captureExperimentalSpanAttributes() {
```

Comment:
Can this use SpringWebFluxConfig?

Reply:
`SpringWebFluxConfig` is in the javaagent module. I didn't want to add public API surface to the library module.